common/lib/xmodule/xmodule/js/spec/problem/edit_spec.coffee

```diff
@@ -191,17 +191,105 @@ describe 'MarkdownEditingDescriptor', ->
       data =  MarkdownEditingDescriptor.markdownToXml("""
         Enter 1 with a tolerance:
         = 1 +- .02
-        or= 2 +- 5%
+        or= 2
         """)
       expect(data).toXMLEqual("""<problem>
         <numericalresponse answer="1">
           <p>Enter 1 with a tolerance:</p>
         <responseparam type="tolerance" default=".02"/>
+        <additional_answer answer="2"/>
           <formulaequationinput/>
         </numericalresponse>
 
 
         </problem>""")
+    it 'markup with multiple additional answers doesn\'t break numerical response', ->
+      data =  MarkdownEditingDescriptor.markdownToXml("""
+        Enter 1 with a tolerance:
+        = 1 +- .02
+        or= 2
+        or= 3
+        """)
+      expect(data).toXMLEqual("""<problem>
+        <numericalresponse answer="1">
+          <p>Enter 1 with a tolerance:</p>
+        <responseparam type="tolerance" default=".02"/>
+        <additional_answer answer="2"/>
+        <additional_answer answer="3"/>
+          <formulaequationinput/>
+        </numericalresponse>
+
+
+        </problem>""")
+    it 'markup with ranged additional answers doesn\'t break numerical response', ->
```

Comment:
I would name it `Do not render ranged additional answers for numerical response'